modules/exploits/firewall/extrabacon.rb

```diff
@@ -222,13 +239,6 @@ def setup
 
     @eb = ExtraBacon.new
     @eb.instance_eval(content)
```

Comment:
instance_eval() can raise exceptions, you probably still want a begin/rescue around this, something like:
```
begin
  @eb.instance_eval(content)
rescue ::Interrupt
  raise $!@
rescue ::Exception
print_error("Failed to load shellcode template for target at #{sc_file}: #{$!}")
end